pkgs/development/python-modules/fairscale/default.nix

```diff
@@ -30,10 +30,13 @@ buildPythonPackage {
     hash = "sha256-L2Rl/qL6l0OLAofygzJBGQdp/2ZrgDFarwZRjyAR3dw=";
   };
 
+  # setup.py dependends on ninja python dependency, but we have the binary in nixpkgs
+  preBuild = ''
```

Comment:
Nit: I more often see this in `postPatch` since I view it as fixing up the source code for nixpkgs.